src/app.test.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable quotes */
+const movies = require("./__mock__/countMovies.js");
+const comments = require("./__mock__/countComment.js");
+
+const firstTestArray = ["a", "b", "c", "d", "e", "f", "g", "h"];
+const secondTestArray = ["a", "b", "c"];
+const thirdTestArray = ["a", "b", "c", "d", "e"];
+
+// Count movies
+
+describe("1. Should test Number of movies", () => {
+  test("First array test", () => {
+    expect(movies(firstTestArray)).toBe(8);
+  });
+  test("Second array test", () => {
+    expect(movies(secondTestArray)).toBe(3);
+  });
+  test("Third array test", () => {
+    expect(movies(thirdTestArray)).toBe(5);
+  });
+});
```

Comment:
- [ ] Kindly remember to cover all edge cases of the method. (like if there's no movies)

Reply:
okey thanks